benchmarks/distributed/rpc/rl/plot_benchmark_report.py

```diff
@@ -0,0 +1,33 @@
+import json
```

Comment:
@mrshenli as discussed this will go in its own repo but wanted to get initial thoughts.  this is to take an input file generated from benchmark run with the graphing feature

Reply:
yep, this looks good to me.